conans/client/graph/graph_manager.py

```diff
@@ -57,23 +58,33 @@ def load_consumer_conanfile(self, conanfile_path, info_folder,
         """
         try:
             graph_info = GraphInfo.load(info_folder)
+            graph_lock_file = GraphLockFile.load(info_folder)
+            graph_lock = graph_lock_file.graph_lock
+            self._output.info("Using lockfile: '{}'".format(info_folder))
```

Comment:
Maybe is better to print with the `conan.lock`

Reply:
Done